programmers-coding-test/kimjiwon/알고리즘/거스름돈.js

```diff
@@ -0,0 +1,32 @@
+function giveChange(customerMoney, goodsPrice) {
+    const unitMoney = [10000 ,5000 ,1000, 500, 100, 50, 10];
+    const changeUnitMoney = new Map();
+
+    let sum = 0,
+    change = customerMoney - goodsPrice;
+
+    unitMoney.forEach(el => {
+        if (calcurate(change, el)) {
+            changeUnitMoney.set(`${el}`, calcurate(change, el));
+            change -= (el * calcurate(change, el)); 
+        }
+    });
+
+    [...changeUnitMoney].forEach(el => {
+        sum += el[1];
+        console.log(`${el[0]}원 : ${el[1]}개` );
+    });
+
+    console.log(`총 ${sum}개`);
+}
+
+
+function calcurate(change, el) {
+    return parseInt(change / el);
+}
```

Comment:
필요한 동전의 개수를 카운팅하는 함수라는 것을 명확히 인지시켜주는 것이 좋을 듯 하다.
```suggestion
function countSpendingMoneys(change, unitMoney) {
    return parseInt(change / unitMoney);
}
```